code/game/objects/items/stunbaton.dm

```diff
@@ -17,6 +17,7 @@
 	var/obj/item/stock_parts/cell/high/cell

```

Comment:
keep in mind cell is `cell/high` here.

Reply:
This should just be changed to be `cell/` and the implicit `new`s updated